src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -100,30 +109,54 @@ public CreateCaseCallbackService(
         ExceptionRecord exceptionRecord,
         ServiceConfigItem configItem,
         long caseId,
-        boolean ignoreWarnings
+        boolean ignoreWarnings,
+        String idamToken,
+        String userId
     ) {
+        String caseIdAsString = Long.toString(caseId);
+
+        log.info("Start creating new case for {} from exception record {}", configItem.getService(), caseIdAsString);
+
         try {
-            log.info(
-                "Start creating exception record for service {} and exception record {}",
-                configItem.getService(),
-                caseId
-            );
+            String s2sToken = s2sTokenGenerator.generate();
 
             SuccessfulTransformationResponse transformationResponse = transformationClient.transformExceptionRecord(
                 configItem.getTransformationUrl(),
                 exceptionRecord,
-                s2sTokenGenerator.generate()
+                s2sToken
             );
 
             if (!ignoreWarnings && !transformationResponse.warnings.isEmpty()) {
                 log.warn("Transformation warnings: {}", String.join(", ", transformationResponse.warnings));
```

Comment:
I know it's not this PR, but worth double checking if we can log warnings from transformation service. @nitinprabhu ?

Reply:
actually warnings may include sensitive Info so it would be safe to not log it

Reply:
gone 🐩